pom.xml

```diff
@@ -225,8 +225,29 @@
     <dependency>
       <groupId>io.fabric8</groupId>
       <artifactId>kubernetes-client</artifactId>
-      <version>4.1.0</version>
+      <version>4.1.3</version>
     </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-core</artifactId>
+      <version>2.9.8</version>
+    </dependency>
+    <dependency>
```

Comment:
why are you adding these as libraries and not plugins?
It has has the potential to break things, see:
https://github.com/jenkinsci/jackson2-api-plugin
https://github.com/jenkinsci/jackson-databind-plugin